Epam/Java Basics_UpSkill Lab #1/Strings_and_basics_of_text_processing/Working_with_a_string_as_a_String_or_StringBuilder/src/com/androsor/string_object/HomeTask4.java

```diff
@@ -0,0 +1,36 @@
+package com.androsor.string_object;
+
+/**
+ * Using the copy function and the concatenation operation, form the word "торт" from the parts of the word "информатика".
+ */
+
+public class HomeTask4 {
+
+    public static void main(String[] args) {
+
+        String str;
+        str = HomeTask1.enterFromConsole("Введите слово или строку");
+
+        System.out.printf("Было: %s \n", str);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.